src/telemetry/index.ts

```diff
@@ -12,6 +12,18 @@ import { isPromise } from 'rxjs/internal-compatibility';
 import { populateTelemetryWithErrorInfo } from '../platform/errors';
 import { IEventNamePropertyMapping } from './types';
 
+/**
+ * TODO@rebornix
```

Comment:
I'll have follow up PRs soon to move these `constants` to right places. `Telemetry` and `VSCodeNativeTelemetry` enums can be moved into `src/telemetry` but `JupyterCommands` can probably be moved to elsewhere.